### PR TITLE
iss6 i18n

### PR DESCRIPTION
**Pull request by annkupriyanova**
There was a problem that the language was set globally, not for each user. I've fixed this.
In commit https://github.com/annkupriyanova/wiki-framework/pull/11/commits/e0cd36edc75d2c593c6c8ece041e580c22c3cee9 new postgres container was also added.

### CODE REVIEW

**Review by orasul — APPROVED** [review]

File: terminology_bot/bot.py [hunk 1]
```diff
@@ -89,44 +83,47 @@ def update_state_handlers(self):
...
         :return: the state START_MENU
         """
         # set language settings, UI-elements and button handlers according User's language_code
         lang_code = update.message.from_user.language_code
-        if lang_code != 'en':
-            change_language(lang_code)
-
-        self.set_keyboard()
-        self.update_state_handlers()
+        print(update.message.from_user.first_name, lang_code)
```
Comment (cdeler) [inline terminology_bot/bot.py]: What does this `print` do?

File: terminology_bot/bot.py [hunk 2]
```diff
@@ -12,9 +12,9 @@
 
 
 # default language settings
-locale_path = 'data/locale/'
-language = gettext.translation('bot', locale_path, ['en'])
-_ = language.gettext
+# locale_path = 'data/locale/'
+# language = gettext.translation('bot', locale_path, ['en'])
+# _ = language.gettext
```
Comment (cdeler) [inline terminology_bot/bot.py]: Please remove out commented lines

File: terminology_bot/bot.py [hunk 3]
```diff
@@ -67,20 +38,43 @@ def __init__(self):
...
+        locale_path = 'data/locale/'
+        language = gettext.translation('bot', locale_path, ['en'])
 
-        self.reply_menu_keyboard = ReplyKeyboardMarkup([[options['pos_tag'], options['description']],
-                                                        [options['synonyms'], options['similars']],
-                                                        [options['image'], options['audio'], options['video']]],
-                                                       resize_keyboard=True)
+        try:
+            language = gettext.translation('bot', locale_path, [lang_code])
+        finally:
```
Comment (cdeler) [inline terminology_bot/bot.py:42]: Looks strange for me. Why do you use
```
try:
   ...
finally:
   ...
``` 
block. Which Exception are you waiting for?

File: terminology_bot/bot.py [hunk 4]
```diff
@@ -89,44 +83,47 @@ def update_state_handlers(self):
...
-        if lang_code != 'en':
-            change_language(lang_code)
-
-        self.set_keyboard()
-        self.update_state_handlers()
+        print(update.message.from_user.first_name, lang_code)
+        lang, options, start_btn, term_btn = self.set_language_and_options(lang_code)
+        user_data.update({'lang': lang, 'options': options, 'start_btn': start_btn, 'term_btn': term_btn})
 
+        _ = user_data['lang'].gettext
```
Comment (cdeler) [inline terminology_bot/bot.py:96]: This line is similar in all method.
Is it ok?

**Review by cdeler — CHANGES_REQUESTED** [review]

File: terminology_bot/bot.py (hunk 4 above)
Comment (annkupriyanova) [reply terminology_bot/bot.py:96]: This is a way to define user specific translation function. Do you know how to make it another way?

File: terminology_bot/bot.py (hunk 1 above)
Comment (annkupriyanova) [reply terminology_bot/bot.py]: I'll remove it. It's for debug.

File: terminology_bot/bot.py (hunk 3 above)
Comment (annkupriyanova) [reply terminology_bot/bot.py:42]: IOError is raised by gettext when there are no translation files needed.

**Review by cdeler — APPROVED** [review]

**Pull request merged**